docs/model_zoo.md

```diff
@@ -6,19 +6,23 @@ To propose a model for inclusion, please submit a [pull request](https://github.
 *Special thanks to the [Apache MXNet](https://mxnet.incubator.apache.org) community whose Model Zoo and Model Examples were used in generating these model archives.*
 
 
-| Model File | Type | Dataset | Size | Download |
-| --- | --- | --- | --- | --- |
-| [CaffeNet](#caffenet) | Image Classification | ImageNet | 216 MB | [.model](https://s3.amazonaws.com/model-server/models/caffenet/caffenet.model) |
-| [Inception v3 w/BatchNorm](#inception) | Image Classification | ImageNet | 45 MB |  [.model](https://s3.amazonaws.com/model-server/models/inception-bn/Inception-BN.model) |
-| [LSTM PTB](#lstm-ptb) | Language Modeling | PennTreeBank | 16 MB | [.model](https://s3.amazonaws.com/model-server/models/lstm_ptb/lstm_ptb.model) |
-| [Network in Network (NiN)](#nin) | Image Classification | ImageNet | 30 MB | [.model](https://s3.amazonaws.com/model-server/models/nin/nin.model) |
-| [ResNet-152](#resnet-152) | Image Classification | ImageNet | 241 MB | [.model](https://s3.amazonaws.com/model-server/models/resnet-152/resnet-152.model) |
-| [ResNet-18](#resnet-18) | Image Classification | ImageNet | 43 MB | [.model](https://s3.amazonaws.com/model-server/models/resnet-18/resnet-18.model) |
-| [ResNet50-SSD](#resnet50-ssd) | SSD (Single Shot MultiBox Detector) | ImageNet | 124 MB | [.model](https://s3.amazonaws.com/model-server/models/resnet50_ssd/resnet50_ssd_model.model) |
-| [ResNext101-64x4d](#resnext101) | Image Classification | ImageNet | 334 MB | [.model](https://s3.amazonaws.com/model-server/models/resnext-101-64x4d/resnext-101-64x4d.model) |
-| [SqueezeNet v1.1](#squeezenet) | Image Classification | ImageNet | 5 MB | [.model](https://s3.amazonaws.com/model-server/models/squeezenet_v1.1/squeezenet_v1.1.model) |
-| [VGG16](#vgg16) | Image Classification | ImageNet | 490 MB | [.model](https://s3.amazonaws.com/model-server/models/vgg16/vgg16.model) |
-| [VGG19](#vgg19) | Image Classification | ImageNet | 509 MB | [.model](https://s3.amazonaws.com/model-server/models/vgg19/vgg19.model) |
+| Model File | Type | Dataset | Source | Size | Download |
+| --- | --- | --- | --- | --- | --- |
+| [AlexNet](#alexnet) | Image Classification | ImageNet | ONNX | 233 MB | [.model](https://s3.amazonaws.com/model-server/models/onnx-alexnet.alexnet.model) |
```

Comment:
onnx-alexnet.alexnet.model ?
should this be slash separated?

Reply:
Doh! Good catch.